relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/stage/actor/BuffManager.kt

```diff
@@ -133,6 +133,13 @@ class BuffManager(val actor: Actor) {
         false
     }
 
+    fun tryRemove(buff: ActiveBuff) = if (buff.turns > 0) {
```

Comment:
This doesn't really make sense. It attempts to remove a specific buff, not a type of buff.